app/livechat/client/views/app/tabbar/customerChatHistoryMessages.html

```diff
@@ -0,0 +1,62 @@
+<template name='customerChatHistoryMessages'>
+    {{#if isLoading}}
+	<h2 class='loding'>
+		{{_ "Loading..."}}
+	</h2>
+	{{else}}
+   
```

Comment:
Please remove all of the blank lines and fix the indentation.

Reply:
done